catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/solr/FilteringSolrIndex.java

```diff
@@ -54,7 +54,7 @@
 
 public class FilteringSolrIndex {
 
-    private static IndexSchema indexSchema;
+    private static volatile IndexSchema indexSchema;
```

Comment:
I took a look at this class and I can't see why we need the memory fence. Then I ran FindBugs locally on the class and it didn't report anything I'd missed. What was the finding that required this change?